lua/arrow/commands.lua

```diff
@@ -0,0 +1,60 @@
+local ui = require("arrow.ui")
+
+local M = {}
+
+function M.cmd(cmd, opts)
+  local command = M.commands[cmd]
+  if command then
+    command()
```

Comment:
we should use opts here, right?

Reply:
I think it was yelling at me since we don't have a command that is using opts yet let me take a look

Reply:
we will just have this warning lua/arrow/commands.lua|8 col 17-21 warning| This function expects a maximum of 0 argument(s) but instead it is receiving 1.

so I'll just an inspectOpts function so no warnings and then you can just get a quick look at the options